packages/yourloops/pages/patient/caregivers/page.tsx

```diff
@@ -161,6 +161,10 @@ function PatientCaregiversPage(props: PatientCaregiversPageProps): JSX.Element {
     }
   }, [caregivers, session, haveNotifications, sentInvitations]);
 
+  React.useEffect(() => {
+    setPageTitle(t("menu-caregivers"));
```

Comment:
`your-loops.com/patient/xxx/caregivers` has to be `My caregivers | YourLoops`according to the spec
```suggestion
    setPageTitle(t("caregivers-title"));
```